docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,67 @@
+
+By comparison, The .NET Framework was built as a completely integrated product, providing end-to-end experiences across Windows, Visual Studio and even SQL Server. Most customers loved that level of integration, but it was difficult to produce meaningful subsets of it. For example, because of API factoring, some subsystems are always required (e.g. Object.GetType has a dependency on Reflection). With .NET Core, those hard API dependencies have been removed and the APIs redefined, to enable the goal of adaptibility.
+
+Native compilation is a fundamental part of the adaptability charter for .NET Core. The .NET team has been working on an experimental toolchain and runtime on GitHub in the [corert](https://github.com/dotnet/corert) repo. It's a derivative of the [.NET Native](https://msdn.microsoft.com/library/dn584397.aspx) project. CoreRT supports multiple native compiler backends, notably an [LLVM-based compiler](https://github.com/dotnet/llilc) and an [IL to C++ compiler](https://github.com/dotnet/corert/tree/master/src/ILCompiler.Compiler/src/CppCodeGen) (meaning, produces textual C++ to use as input to a C++ compiler). The team will continue to invest in CoreRT to make it a production option for .NET Core apps. This will enable .NET to run on any platform that supports a C++ compiler and/or LLVM (e.g. WebAssembly).
+
+Another advantage of CoreRT is that most of the runtime services have been moved to C# libraries, making them also opt-in. The CoreRT runtime is so small (<1mb) that it is fine that it is required in its complete form. Over time, we expect that most scenarios will adopt CoreRT given that it will be the easiest to adapt to new scenarios. Also, since it implements fewer concepts in the runtime, it's also an easier codebase to learn, making it accessible to a much larger group of developers.
+
+Comparison with .NET Framework
+------------------------------
+
+The .NET platform was first announced by Microsoft in 2000 and then evolved from there, largely focused on the .NET Framework product. For example, the Mono runtime is an open and cross-platform implementation of the .NET Framework. .NET Core continues on the promise and tradition of the .NET Framework, and conforms to the same [.NET standards](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) (notably ECMA 335), but is a separate product. 
```

Comment:
The Mono tidbit seems odd and out of place. Also, what are the promises you mention?